src/OrchardCore.Build/Dependencies.props

```diff
@@ -39,7 +39,7 @@
     <PackageManagement Include="NodaTime" Version="3.1.0" />
     <PackageManagement Include="OpenIddict.AspNetCore" Version="3.1.1" />
     <PackageManagement Include="OpenIddict.Core" Version="3.1.1" />
-    <PackageManagement Include="OrchardCore.Translations.All" Version="1.3.0" />
+    <PackageManagement Include="OrchardCore.Translations.All" Version="1.0.1-beta-20019" />
```

Comment:
beta?

Reply:
We are testing the translations package before publishing it as 1.4 on nuget.org.